assets/js/components/adminbar/AdminBarApp.js

```diff
@@ -75,7 +80,36 @@ export default function AdminBarApp() {
 						mdc-layout-grid__cell--align-middle
 					">
 						<div className="mdc-layout-grid__inner">
-							<AdminbarModules />
+							{ featureFlags.widgets.adminBar.enabled
+								? (
+									<Fragment>
+										{
+											( analyticsModuleActive && analyticsModuleConnected )
+												? (
+													<Fragment>
+														<AdminBarUniqueVisitors />
+														<AdminBarSessions />
+													</Fragment>
+												)
+												: (
+													<div className="
+														mdc-layout-grid__cell
+														mdc-layout-grid__cell--span-6-desktop
+														mdc-layout-grid__cell--span-4-tablet
+													">
+														<AnalyticsInactiveCTA />
+													</div>
+												)
+										}
```

Comment:
Let's simplify this to be less nested using basic conditions rather than ternary.
```js
{ analyticsModuleConnected && (
	<Fragment>
		<AdminBarUniqueVisitors />
		<AdminBarSessions />
	</Fragment>
) }
{ ! analyticsModuleConnected && (
	<div className="
		mdc-layout-grid__cell
		mdc-layout-grid__cell--span-6-desktop
		mdc-layout-grid__cell--span-4-tablet
	">
		<AnalyticsInactiveCTA />
	</div>
) }
```

Reply:
Updated @aaemnnosttv . Any reason why we don't use ternary? Is it because of performance reasons?

Reply:
I think it's mostly our convention and a preference that makes the code a bit easier to follow, especially with larger blocks. In this case it results in substantially less indentation/nesting. It also makes it a bit easier to see the logic for each rendered chunk whereas with a ternary you have to kind of backtrack to the original conditional and mentally flip the logic. We don't avoid it in all cases, but we generally prefer not to use it in JSX.